Sources/PrimerSDK/Classes/Data Models/Theme/Internal/PrimerTheme+Colors.swift

```diff
@@ -0,0 +1,53 @@
+public extension UIColor {
+
+    static func primer(lightMode: UIColor, darkMode: UIColor) -> UIColor {
```

Comment:
[Feedback]
I don't believe this is a good name for a function that returns one of the two provided colors.

Reply:
Yeah, agree. Debating whether we should actually have it. Technically, if the merchant wants to pass in a UIColor that dynamically switches between dark mode and light mode, they can do so on their end. Wdyt?